salt/metalk8s/kubeadm/init/certs/front-proxy-ca.sls

```diff
@@ -0,0 +1,77 @@
+{%- from "metalk8s/map.jinja" import front_proxy_ca with context %}
+
+{%- set front_proxy_ca_server = salt['mine.get']('*', 'kubernetes_front_proxy_ca_server') %}
+
+{#- Check if we have no CA server or only current minion as CA #}
+{%- if not front_proxy_ca_server or front_proxy_ca_server.keys() == [grains['id']] %}
+
+include:
+  - .installed
+  - metalk8s.salt.minion.running
+
+Create front proxy CA private key:
+  x509.private_key_managed:
+    - name: /etc/kubernetes/pki/front-proxy-ca.key
+    - bits: 4096
+    - user: root
+    - group: root
+    - mode: 600
+    - makedirs: True
+    - dir_mode: 755
+    - require:
+      - pkg: Install m2crypto
+
+Generate front proxy CA certificate:
+  x509.certificate_managed:
+    - name: /etc/kubernetes/pki/front-proxy-ca.crt
+    - signing_private_key: /etc/kubernetes/pki/front-proxy-ca.key
+    - CN: front-proxy-ca
+    - keyUsage: "critical digitalSignature, keyEncipherment, keyCertSign"
+    - basicConstraints: "critical CA:true"
+    - days_valid: {{ front_proxy_ca.cert.days_valid }}
+    - user: root
+    - group: root
+    - mode: 644
+    - makedirs: True
+    - dir_mode: 755
+    - require:
+      - x509: Create front proxy CA private key
+
+# TODO: Find a better way to advetise CA server
+Advertise front proxy CA in the mine:
+  module.wait:
+    - mine.send:
+      - func: 'kubernetes_front_proxy_ca_server'
+      - mine_function: hashutil.base64_encodefile
+      - /etc/kubernetes/pki/front-proxy-ca.crt
+    - watch:
+      - x509: Generate front proxy CA certificate
+
+Create front proxy CA salt signing_policies:
+  file.serialize:
+    - name: /etc/salt/minion.d/30-metalk8s-ca-signing-policies.conf
```

Comment:
suggest to use `/etc/salt/minion.d/30-metalk8s-front-proxy-ca-signing-policies.conf` please